cluster/manifests/flannel/daemonset.yaml

```diff
@@ -36,29 +35,6 @@ spec:
             cpu: 1m
             memory: 50Mi
       containers:
-      - name: delayed-install-cni
-        image: container-registry.zalando.net/teapot/flannel-awaiter:master-9
-        command:
-        - /await-and-copy
```

Comment:
The awaiter doesn't only copy the config, it also checks (and waits) that it can reach _all_ other nodes in the cluster. Without this flannel could become ready before it truly is ready.

We can only drop the _copy config_ part, not the _await_ part.

Reply:
Oh, I understood our discussion as we don't need the _await_ anymore because of `kube-node-ready`. I see. I'll bring back the container. Thanks!

Reply:
We might need to patch the `flannel-awaiter` to remove the copy functionality from the script and only keep the `await` part 🤔

Reply:
Since `flannel-awaiter` has been updated [here](https://github.bus.zalan.do/teapot/flannel-awaiter/pull/10), will proceed to add the container back with the updated version and desired readiness Probe.

Reply:
I've added the awaiter container back with the startup probe to probe the dummy file.

Reply:
The probe needs fixing. Working on that.

Reply:
Fixed the probe.